draft-ietf-suit-manifest.md

```diff
@@ -1493,7 +1493,9 @@ Expert reviewers should take into consideration the following points:
       length of the encoded value should be weighed against how many
       code points of that length are left, the size of device it will be
       used on, and the number of code points left that encode to that
-      size.
+      size. For each registry, values 0-255 are Standards Action and 
```

Comment:
The expert can read.

Copying information from the registry definition to the DE instructions is unnecessary and error-prone; it wastes the time of everybody who now needs to check that these clones do not conflict.

Reply:
I have removed this line